src/unit_tests/remoted/test_sendmsg.c

```diff
@@ -39,32 +39,166 @@ static int group_teardown(void ** state) {
     return 0;
 }
 
-static int test_setup_tcp(void ** state) {
+static int test_setup_keys(void ** state) {
     test_mode = 1;
 
     keyentry *node_key = NULL;
     os_calloc(1, sizeof(keyentry), node_key);
-    node_key->net_protocol = REMOTED_NET_PROTOCOL_TCP;
     node_key->rcvd = 10;
     node_key->sock = 15;
+    node_key->id = strdup("001");
```

Comment:
We usually use os_strdup() or w_strdup().

Reply:
Right! Fixed.